usr.sbin/kldxref/ef.c

```diff
@@ -636,6 +685,7 @@ ef_open(struct elf_file *efile, int verbose)
 static void
 ef_close(elf_file_t ef)
 {
+	free(ef->ef_capreloc);
```

Comment:
I assume ef gets calloc/memset'ed so the lack of initialisation for ef_capreloc when it doesn't exist is fine?

Reply:
Yes, same for `ef_rel` vs `ef_rela` that typically only one of is allocated but not both.